lib/fastlane/plugin/wpmreleasetoolkit/helper/metadata_download_helper.rb

```diff
@@ -16,12 +16,7 @@ def initialize(target_folder, target_files)
       def download(target_locale, glotpress_url, is_source)
         uri = URI(glotpress_url)
         response = Net::HTTP.get_response(uri)
```

Comment:
Just a thought, not related to this PR: it might be good to use faraday which has features like following redirects and retry with an interval (which could be a replacement for handling 429 error?).

Reply:
Yeah... I don't know about [faraday](https://lostisland.github.io/faraday/usage/) or other Ruby networking clients, but given the number of network requests we do with this toolkit, having a feature-rich way to do requests would be handy.

Reply:
iinm Farady is also the network client that fastlane itself uses, so it should already be part of our ruby dependencies anyway.

This is one of those items part of a long list of little things I'd love to improve / refactor to clean up the release-toolkit in many areas and modernize it… but never got the bandwidth to 😅